src/bus.rs

```diff
@@ -115,19 +115,31 @@ where
 		}
 	}
 
-	/// Write a raw instruction to a stream, and read a single raw response.
-	///
-	/// This function also checks that the packet ID of the status response matches the one from the instruction.
-	///
-	/// This is not suitable for broadcast instructions.
-	/// For broadcast instructions, each motor sends an individual response or no response is send at all.
-	/// Instead, use [`Self::write_instruction`] and [`Self::read_status_response`].
-	pub fn transfer_single<F>(
-		&mut self,
-		packet_id: u8,
-		instruction_id: u8,
-		parameter_count: usize,
-		encode_parameters: F,
+    /// Get a reference to the underlying [`SerialPort`].
+    pub fn stream(&self) -> &SerialPort {
```

Comment:
```suggestion
    pub fn serial_port(&self) -> &SerialPort {
```

I used the term "stream" originally because it was based on generic `Read` and `Write` implementations. But now the serial port implementation is fixed, so we can be more explicit.

The documentation still says "stream" in most places, so I understand why you chose this name. But I prefer to reword the documentation instead. No need to do that in this PR though, I can do that part.